tools/system_libs.py

```diff
@@ -1278,6 +1278,8 @@ def can_build(self):
 class libwasmfs(MTLibrary):
   name = 'libwasmfs'
 
+  cflags = ['-fno-exceptions']
```

Comment:
Can you split this out into a separate tiny PR so we can make sure that everyone agrees this makes sense?